### PR TITLE
Rename AlwaysOn to Mission-Critical

### PR DESCRIPTION
**Pull request by heoelri**
This PR is intended to replace the former name **AlwaysOn** with **Azure Mission-Critical** respective **Mission-Critical**.

It does not update the links to the online and connected RIs nor the application.

# ToDo

* Update application
* Rename RI repositories
* Update links to RI repositories
* Update diagrams

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 
 ## Design Methodology
 
-> I need to design a highly reliable and mission-critical application on Azure. Where can I learn more about AlwaysOn design methodology so?
+> I need to design a highly reliable and mission-critical application on Azure. Where can I learn more about Azure Mission-Critical design methodology so?
 
-The AlwaysOn design principles and design areas are published within this repository and you can learn more about how to use the design methodology [here](/docs/design-methodology/README.md). 
-The AlwaysOn design methodology is also available within the [Microsoft Azure Architecture Center](https://docs.microsoft.com/azure/architecture/mission-critical/alwayson-overview) for general consumption.
+The Azure Mission-Critical design principles and design areas are published within this repository and you can learn more about how to use the design methodology [here](/docs/design-methodology/README.md).
+The Azure Mission-Critical design methodology is also available within the [Microsoft Azure Architecture Center](https://docs.microsoft.com/azure/architecture/mission-critical/alwayson-overview) for general consumption.
```
Comment (heoelri) [inline FAQ.md]: @CalCof the link to the Azure Architecture center is broken.

**Comment by msimecek** [comment]
Is this the official direction for us? Where was it discussed? I thought we would be keeping AlwaysOn for the reference implementation and use Mission-Critical for the Docs (design methodology) stuff.

File: FAQ.md (hunk 1 above)
Comment (hjscherer) [reply FAQ.md]: @heoelri link is updated

**Review by hjscherer — APPROVED** [review]
LGTM @CalCof

**Review by CalCof — APPROVED** [review]

**Pull request merged**